tools/dcgm_api_export.h

```diff
@@ -0,0 +1,14 @@
+#ifndef DCGM_DCGM_API_EXPORT_H
```

Comment:
This is an entire file, not a patch. It is less safe to overwrite the entire file than to just apply a patch. The patch will fail if the file changes and lead to a failed build which will allow us to notice that something changed. Also this should live in "build/libgcgm/dcgm_api_export.patch". See onnxruntime_backend for example of applying the patch.

Reply:
@deadeyegoodwin we're just adding a new file (it's not actually a patch). Libdcgm does not currently have a release with this change. Should we just wait for them?

Reply:
The addition of this file can be the patch. I agree with @deadeyegoodwin that it should be in `build/libgcgm/dcgm_api_export.patch` and that you should apply this patch. Refer to how we do this in our ORT backend.

Reply:
We talked about this offline. With the above build.py changes copying a complete file is ok. But it still needs to live in build/libdcgm